xml/System.Reflection.Emit/AssemblyBuilder.xml

```diff
@@ -64,52 +64,53 @@
 
-The signing of a dynamic assembly using <xref:System.Reflection.AssemblyName.KeyPair%2A> is not effective until the assembly is saved to disk. So, strong names will not work with transient dynamic assemblies. 
+The signing of a dynamic assembly using <xref:System.Reflection.AssemblyName.KeyPair%2A> is not effective until the assembly is saved to disk. So, strong names will not work with transient dynamic assemblies.
 
 Dynamic assemblies can reference types defined in another assembly. A transient dynamic assembly can safely reference types defined in another transient dynamic assembly, a persistable dynamic assembly, or a static assembly. However, the common language runtime does not allow a persistable dynamic module to reference a type defined in a transient dynamic module. This is because when the persisted dynamic module is loaded after being saved to disk, the runtime cannot resolve the references to types defined in the transient dynamic module.
 
 ### Restrictions on emitting to remote application domains in .NET Framework
- Some scenarios require a dynamic assembly to be created and executed in a remote application domain. Reflection emit does not allow a dynamic assembly to be emitted directly to a remote application domain. The solution is to emit the dynamic assembly in the current application domain, save the emitted dynamic assembly to disk, and then load the dynamic assembly into the remote application domain. The remoting and application domains are supported only in .NET Framework. 
-  
-## Examples  
+ Some scenarios require a dynamic assembly to be created and executed in a remote application domain. Reflection emit does not allow a dynamic assembly to be emitted directly to a remote application domain. The solution is to emit the dynamic assembly in the current application domain, save the emitted dynamic assembly to disk, and then load the dynamic assembly into the remote application domain. The remoting and application domains are supported only in .NET Framework.
+
+## Examples
 The following code example shows how to define and use a dynamic assembly. The example assembly contains one type, `MyDynamicType`, that has a private field, a property that gets and sets the private field, constructors that initialize the private field, and a method that multiplies a user-supplied number by the private field value and returns the result.
 
  :::code language="cpp" source="~/snippets/cpp/VS_Snippets_CLR/AssemblyBuilderClass/cpp/24895.cpp" id="Snippet1":::
  :::code language="csharp" source="~/snippets/csharp/System.Reflection.Emit/AssemblyBuilder/Overview/24895.cs" id="Snippet1":::
- :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR/AssemblyBuilderClass/vb/24895.vb" id="Snippet1":::  
-  
+ :::code language="fsharp" source="~/snippets/fsharp/System.Reflection.Emit/AssemblyBuilder/Overview/24895.fs" id="Snippet1":::
```

Comment:
You referenced a Snippet1, but didn't define an actual named snippet in source (which is done by syntax like `//<Snippet1> ... //</Snippet1>`). Consider dropping `id="Snippet1"` part if you want to just reference a whole source file.

Reply:
Thanks a lot! Fixed it.